### ISSUE TITLE
No playback using VPN (Arch Linux)

### ISSUE THREAD

**Issue by nuehm-arno** [FreeTubeApp/FreeTube#1128]
**Behavior of the program**
When using a VPN, it is not possible to play any video. See additional context for error logs.
Searching for videos is still possible.

**Expected behavior**
Playback of videos using VPN.

**To Reproduce**
0. Start VPN connection (RiseupVPN)
1. Go to 'search' and search for artist/video
2. Click on 'video'
3. No video playback
4. See error logs

**Environment Information (please complete the following information):**
 - OS: Arch Linux
 - OS Version: up-to-date
 - FreeTube version: v0.12.0 Beta (AUR)
 - Installation Method (If applicable): AUR
 - Primary API Used: Local API

**Additional context**
Local API Error: Status code: 429
Invidious API Error: undefined

**Comment by TheSunnyWolf** [comment]
status code 429 occur when the server throttles you.
to put it simply, youtube blocked that IP due to too many requests, it's a temporary block that last (usually) for less than 24 hours.
since you are using a VPN it's likely caused by multiple users sharing the same IP.

I would recommend closing this issue, as it's not related to freetube.

**Comment by nuehm-arno** [comment]
It seems, my VPN is blocked on Google/Youtube for a longer period of time... So wouldn't it be nice to have the Google captcha popping up, if an IP is blocked to "free" it?

**Comment by nuehm-arno** [comment]
Closing, but a captcha option would be nice to have.
